src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -25,6 +27,10 @@
                                                       'aria-current': 'true')
 %p
   - if submit_actions_count > 1
+    - if User.session
+      .d-inline#action-seen-by-user-wrapper
+        = render ActionSeenByUserComponent.new(action: active_action, user: User.session!)
+
     - active_action_index = submit_actions.find_index(active_action) + 1
     Showing
     %em action ##{active_action_index}
```

Comment:
I'd propose a little change here to improve alignment and spacing:

```   
    %span.ms-2.align-middle
      Showing
      %em action ##{active_action_index}
      \-
  %span.font-italic.align-middle= request_action_header(action, bs_request.creator)    
```

(sorry, GH doesn't allow me to add a "Suggestion" beyond the lines you modified.) 

What I propose is:
- Wrap the "Showing action #N" part with a `span` which adds `.ms-2.align-middle`
- Add `.align-middle` to the `request_action_header` as well.


**Before:**

![Screenshot 2023-01-23 at 10-42-41 Open Build Service](https://user-images.githubusercontent.com/2581944/214008816-8a208e74-e7e1-4486-b46a-e0c3531b914f.png)

**After:**

![Screenshot 2023-01-23 at 10-45-59 Open Build Service](https://user-images.githubusercontent.com/2581944/214009303-7727c76c-9dd0-4d73-8b03-2cecc4615fb2.png)